test/common/upstream/health_check_fuzz_test_utils.h

```diff
@@ -48,6 +48,7 @@ class HttpHealthCheckerImplTestBase : public HealthCheckerTestBase {
     Network::MockClientConnection* client_connection_{};
     NiceMock<Http::MockRequestEncoder> request_encoder_;
     Http::ResponseDecoder* stream_response_callbacks_{};
+    CodecClientForTest* codec_client_{};
```

Comment:
Are these changes needed?

Reply:
yeah, I have to capture the codec_client_ because that's what I call `raiseGoAway(_)` on

Reply:
Oh I see sorry didn't realize this was used elsewhere.